imports/ui/components/profile/profile_edit.html

```diff
@@ -1,7 +1,7 @@
 <template name='profileEdit'>
   <div class='profileEdit'>
   {{#if currentUser}}
-    {{> quickForm id="profileEdit" collection="Meteor.users" doc=currentUser type="update" fields="profile"}}
+    {{> quickForm id="profileEdit" collection=collection doc=currentProfile type="update" omitFields="userId"}}
```

Comment:
I knew we were getting something wrong haha. `collection=collection` is such a silly looking bit of syntax. Glad you got this working :)